internal/wasm/store.go

```diff
@@ -172,6 +181,10 @@ type (
 		Max   *uint32
 		// Currently fixed to 0x70 (funcref type).
 		ElemType byte
+
+		// address is the tableaddr(https://www.w3.org/TR/wasm-core-1/#syntax-tableaddr) of this table instance.
+		// In other words, this equals the index of this table instance in store.Tables.
```

Comment:
probably better to call this `index` and use the existing type `wasm.Index` vs the confusing uint64 which is at best an implementation detail of additions.

```suggestion
		// address is the index of this table instance in Store.Tables.
		// See https://www.w3.org/TR/2019/REC-wasm-core-1-20191205/#syntax-tableaddr
```